inst/tests/tests.Rraw

```diff
@@ -18315,3 +18315,37 @@ if (test_bit64) {
   apple = data.table(id = c("a", "b", "b"), time = c(1L, 1L, 2L), y = i64v[1:3])
   test(2248, dcast(apple, id ~ time, value.var = "y"), data.table(id = c('a', 'b'), `1` = i64v[1:2], `2` = i64v[4:3], key='id'))
 }
+
+# trunc.char misbehaves for multibyte characters, #5096
+# Tests for combining characters
+options(datatable.prettyprint.char=4L)
+DT = data.table("\u0061\u0301\u0061\u0301\u0061\u0301\u0061\u0301")
+test(2248.01, print(DT), output="1: \u0061\u0301\u0061\u0301\u0061\u0301\u0061\u0301")
+options(datatable.prettyprint.char=3L)
+test(2248.02, print(DT), output="1: \u0061\u0301\u0061\u0301\u0061\u0301...")
+options(datatable.prettyprint.char=1L)
+test(2248.03, print(DT), output="1: \u0061\u0301...")
+# Tests for full-width multibyte characters
+options(datatable.prettyprint.char=4L)
+DT = data.table("\u4E00\u4E8C\u3053\u3093")
```

Comment:
```suggestion
# Japanese characters 'one' 'two' 'ko' 'n'
DT = data.table("\u4E00\u4E8C\u3053\u3093")
```

Reply:
Thanks for the changes, was just about to ask about test readability.